data/json/items/melee/swords_and_blades.json

```
@@ -708,6 +978,111 @@
     "weapon_category": [ "KNIVES" ],
     "melee_damage": { "bash": 3, "cut": 23 }
   },
+  {
+    "id": "lc_kukri",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "low carbon steel kukri" },
+    "description": "This versatile implement is a custom-forged version of a traditional weapon that originated in Nepal.  Featuring a heavy blade with an inwardly-curved edge, this one is made of low carbon steel, and barely has a proper handle, you'd probably find a better version in a store somewhere.",
+    "weight": "450 g",
+    "volume": "750 ml",
+    "longest_side": "50 cm",
+    "price": 8000,
+    "price_postapoc": 1250,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "good" },
```

```suggestion
    "to_hit": { "grip": "solid", "length": "short", "surface": "line", "balance": "good" },
```
"barely has a proper handle"